src/System.Text.Json/src/System/Text/Json/JsonReaderState.cs

```diff
@@ -49,8 +49,9 @@ public struct JsonReaderState
         /// </summary>
         /// <param name="maxDepth">Sets the maximum depth allowed when reading JSON, with the default set as 64.
         /// Reading past this depth will throw a <exception cref="JsonReaderException"/>.</param>
-        /// <param name="commentHandling">Defines how the <see cref="Utf8JsonReader"/> should handle comments when reading through the JSON.
-        /// By default, the <see cref="Utf8JsonReader"/> treats comments within the JSON as invalid.</param>
+        /// <param name="options">Defines the customized behaviour of the <see cref="Utf8JsonReader"/>
```

Comment:
`behaviour` => `behavior` (en-US spelling)

Reply:
K, force of `Canadian` habit. :)